src/tests/src/libs/antares/study/test_study.cpp

```diff
@@ -199,6 +199,16 @@ BOOST_FIXTURE_TEST_CASE(thermal_cluster_delete, ThermalClusterStudy)
     BOOST_CHECK(areaA->thermal.list.empty());
 }
 
+BOOST_FIXTURE_TEST_CASE(thermal_cluster_forceNoGen, ThermalClusterStudy)
```

Comment:
Can we find a better name for this test to better describe the behaviour we want to validate.
Maybe : WithForceNoGenOptionTimeSeriesNotGeneratedForReverseSpinning

Reply:
_